features/support/form_helpers.rb

```diff
@@ -15,6 +15,18 @@ def fill_in_field(field_name, value)
     end
   end
 
+  def javascript_to_simulate_paste(element_id, html_to_paste)
```

Comment:
```suggestion
  def javascript_to_simulate_paste(element_id, paste_content)
```

Minor: there isn't really any need for this function to reference html in the argument